contracts/multisig/src/types.rs

```diff
@@ -44,7 +45,7 @@ impl TryFrom<HexBinary> for MsgToSign {
             });
         }
 
-        Ok(MsgToSign::unchecked(other))
+        Ok(MsgToSign(other))
```

Comment:
why change this?

Reply:
`MsgToSign::unchecked` is now only defined for tests. So, this needs to be constructed directly.